app.js

```diff
@@ -183,6 +192,42 @@ sync.connect(mongodbConnectionString, mongoOptions, redisUrl, function startAppl
     });
   });
 
+  app.get('/metrics', function (req, res) {
+    function getOrMakeHistogram(name, help){
+      if(histograms.hasOwnProperty(name)) {
```

Comment:
Do we need to use our own store here? I think the promClient.registry also stores everything in memory?

https://github.com/siimon/prom-client/blob/master/lib/registry.js#L96

I may be misreading that though.

Reply:
nice spot, I recall you mentioning that earlier alright. I've updated this now.